app/livechat/server/lib/Livechat.js

```diff
@@ -876,6 +900,7 @@ export const Livechat = {
 				status,
 			});
 		});
+		callbacks.runAsync('livechat.agentStatusChanged', { userId, status });
```

Comment:
this callback doesn't depend on the `'Livechat_show_agent_info` setting, so I suggest to run the callback at the beginning of the method, before testing the `Livechat_show_agent_info`  setting.